go.mod

```diff
@@ -16,17 +16,16 @@ require (
 	github.com/coreos/go-iptables v0.6.0
 	github.com/emicklei/go-restful/v3 v3.9.0
 	github.com/evanphx/json-patch v5.6.0+incompatible
-	github.com/go-logr/stdr v1.2.2
 	github.com/greenpau/ovsdb v1.0.2
 	github.com/k8snetworkplumbingwg/network-attachment-definition-client v1.3.0
+	github.com/kubeovn/libovsdb v0.0.0-20221123021657-3e352fe18263
```

Comment:
we can use go mod replace to point to a new path and avoid all the import rename